dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionNameFormatJsonConverter.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// A custom JSON converter for converting function names in a JSON array.
+/// This converter replaces dots used as a function name separator in prompts with hyphens when reading and back when writing.
+/// </summary>
+[Experimental("SKEXP0001")]
+public sealed class FunctionNameFormatJsonConverter : JsonConverter<IList<string>>
```

Comment:
nit

Some providers may use other separators, not sure where this will be used, just by our abstractions?

We might consider passing what is the separator as an init ?

```suggestion
public sealed class FunctionNameFormatJsonConverter : JsonConverter<IList<string>>
```

Reply:
The converted has been removed.